datamodel/changelogs/0001/51_aquifier_2015_dictionaries.sql

```diff
@@ -0,0 +1,26 @@
+ ------ this file generates the tww is_dictionary (Table _aquifier dss 2015) in en on TEKSI
+------ For questions etc. please contact Stefan Burckhardt stefan.burckhardt@sjib.ch
+------ version 9.2.2024 09:18:00
+------ // skip-keyword-check based on version 04.07.2017 23:28:49 09_qgep_dictionaries.sql
+------ with 3D coordinates
+
+--- CREATE TABLE tww_sys.dictionary_od_table already exists
+
+ INSERT INTO tww_sys.dictionary_od_table (id, tablename, name_en, shortcut_en, name_de, shortcut_de, name_fr, shortcut_fr, name_it, shortcut_it, name_ro, shortcut_ro) VALUES (30,'_aquifier','aquifier','AQ','Grundwasserleiter','GL','Aquifère','AQ','NULL','NULL','NULL','NULL');
```

Comment:
you always use 30 as value for id.

Reply:
Each table has a unique id - that is correct. But I added wrongly the same line in tww_sys.dictionary_od_values